src/histolab/tiler.py

```diff
@@ -1,17 +1,18 @@
 from abc import abstractmethod
-from typing import Tuple
+from typing import List, Tuple
 
 import numpy as np
-import sparse
 
+from histolab.exceptions import LevelError
```

Comment:
I do have a concern about this import. I'd prefer relative imports also because we could have some issues with the coverage report.